monai/handlers/mlflow_handler.py

```diff
@@ -72,11 +76,21 @@ class MLFlowHandler:
         state_attributes: expected attributes from `engine.state`, if provided, will extract them
             when epoch completed.
         tag_name: when iteration output is a scalar, `tag_name` is used to track, defaults to `'Loss'`.
+        experiment_name: name for an experiment, defaults to `default_experiment`.
+        run_name: name for run in an experiment.
+        experiment_param: a dict recording parameters which will not change through whole experiment,
+            like torch version, cuda version and so on.
+        artifacts: paths to images that need to be recorded after a whole run.
+        optimizer_param_names: parameters' name in optimizer that need to be record during runing,
+            defaults to `["lr"]`.
 
     For more details of MLFlow usage, please refer to: https://mlflow.org/docs/latest/index.html.
 
     """
 
+    default_log_param_list = ["network", "device", "optimizer", "loss_function"]
```

Comment:
I don't get what we will log for these objects?

Thanks.

Reply:
Hi @Nic-Ma 
It tries to log the name of these objects' type to make users check their basic workflow easily. For example, if UNet were used, it will log `network:<class 'monai.networks.nets.unet.UNet'>`.  As we will log all the meta info, maybe these infos are not necessary. What would you think?
Thanks,
Bin

Reply:
I would suggest to remove this feature directly.

Thanks.